src/Libraries/RevitNodes/Application/Document.cs

```diff
@@ -62,6 +66,71 @@ public static Document Current
             get { return new Document(DocumentManager.Instance.CurrentDBDocument); }
         }
 
+        /// <summary>
+        /// Purge unused Elements from the model. This node is not able to purge materials and material assets
+        /// </summary>
+        /// <param name="deepPurge">Similar to clicking the purge button multiple times in Revit.</param>
+        /// <returns></returns>
+        public string PurgeUnused(bool deepPurge)
+        {
+            //The internal GUID of the Performance Adviser Rule 
+            const string purgeGuid = "e8c63650-70b7-435a-9010-ec97660c1bda";
+
+            var performanceAdviserRuleId = new List<PerformanceAdviserRuleId>();
+            var performanceAdviserRules = PerformanceAdviser.GetPerformanceAdviser().GetAllRuleIds();
+
+            //Iterating through all PerformanceAdviser rules looking to find that which matches PURGE_GUID
+            for (int i = 0; i < performanceAdviserRules.Count; i++)
+            {
+                if (performanceAdviserRules[i].Guid.ToString() == purgeGuid)
+                {
+                    performanceAdviserRuleId.Add(performanceAdviserRules[i]);
+                    break;
+                }
+            }
+
```

Comment:
replace this with a LINQ query `.Where(string.AreEqual(x=>x.Guid, purgeGuid).ToList()`
also update comment to say something like `retrieve {ruleNameHere} rule from list of available PerformanceAdviserRules`. we then need to check that we did actually retrieve/find anything before continuing

also, this will only ever return 1 (the first) id that matches, so no need to store it into a list, use a `var` and this all becomes a one-liner.